src/FakeItEasy/Creation/CastleDynamicProxy/CastleDynamicProxyInterceptionValidator.cs

```diff
@@ -37,7 +37,11 @@ public bool MethodCanBeInterceptedOnInstance(MethodInfo method, object? callTarg
 #else
                 var explicitImplementation = method.Name.Contains('.');
 #endif
-                if (!explicitImplementation)
+                if (explicitImplementation)
+                {
+                    return "The base type implements this interface method explicitly. In order to be able to intercept this method, the fake must specify that it implements this interface in the fake creation options.";

```

Comment:
The message doesn't mention the the actual types and methods involved, for two reasons:
- It would be difficult to get all the required pieces of information here (we already get them at some other point, but passing them to this point would require many changes, and finding them again would be expensive)
- The other messages returned by this method are all generic, so it would be inconsistent to make this one specific. The full error message contains the necessary information anyway.